.gitlab/pipeline/check.yml

```diff
@@ -90,6 +90,16 @@ check-rust-feature-propagation:
     - cargo install --locked --version 0.13.3 -q -f zepter && zepter --version
     - zepter run check
 
+check-toml-format:
+  stage: check
+  extends:
+    - .kubernetes-env
+    - .common-refs
+  script:
+    - cargo install taplo-cli --locked --version 0.8.1
+    - rm -rf polkadot/zombienet_tests # this dir contains invalid toml
```

Comment:
If we keep the check like this I'd suggest creating an issue for this folder and adding the link here as a comment.

Reply:
Created an `exclude` in the taplo config instead.

The invalid toml seems to be some deliberate (custom?) templating syntax, I'll leave it.